client/my-sites/domains/domain-management/edit/wpcom-domain.jsx

```diff
@@ -83,7 +88,7 @@ class WpcomDomain extends React.Component {
 				{ this.getEditSiteAddressBlock() }
 			</div>
 		);
-		/* eslint-disable wpcalypso/jsx-classname-namespace */
+		/* eslint-enable wpcalypso/jsx-classname-namespace */
```

Comment:
Was this toggle intentional?

Reply:
Yes. This component got refactored while I was working on this and I think this was a mistake.